nn/math_.py

```diff
@@ -41,6 +41,11 @@ def floor(x: nn.Tensor) -> nn.Tensor:
   return _activation(x, activation="floor")
 
 
+def rint(x: nn.Tensor) -> nn.Tensor:
+  """round"""
```

Comment:
The docstring is misleading. rint is not round, although very similar. round is another separate function.

Reply:
oh true, had it on round first. Will change